<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetDateTimeForm</class>
 <widget class="QWidget" name="GetDateTimeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор даты и времени</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_message">
     <property name="text">
      <string>Выберите дату и время для напоминания</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="cw_date"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="te_time"/>
   </item>
   <item>
    <widget class="QPushButton" name="pb_done">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
